metis-dereference/metis-dereference-rest/src/main/java/eu/europeana/metis/dereference/rest/controller/DereferencingController.java

```diff
@@ -50,14 +48,7 @@ public DereferencingController(DereferenceService dereferenceService) {
   @GetMapping(value = RestEndpoints.DEREFERENCE, produces = {MediaType.APPLICATION_JSON_VALUE,
       MediaType.APPLICATION_XML_VALUE})
   @ResponseBody
-  @Operation(description = "Dereference a URI",
-      responses = {
-          @ApiResponse(responseCode = "200", content = {
-              @Content(
-                  mediaType = MediaType.APPLICATION_JSON_VALUE,
-                  schema = @Schema(implementation = EnrichmentResultList.class))
-          }),
-      })
+  @Operation(description = "Dereference a URI", responses = {@ApiResponse(responseCode = "200")})
```

Comment:
Shouldn't we write to return results both in XML and JSON? Or like this it returns both by default?
The same comment applies for the rest of the changes

Reply:
Yes indeed, if not mentioned it captures what it reads from the other annotations(as before essentially).